src/lib.rs

```diff
@@ -68,6 +68,13 @@ impl GptConfig {
         cfg
     }
 
+    /// Size of logical blocks (sectors) for this disk.
+    pub fn logical_block_size(self, lb_size: disk::LogicalBlockSize) -> Self {
```

Comment:
Why does this return Self instead of taking an &mut self?  I generally find things that return Self more difficult to use in practice.

Reply:
:+1:

Reply:
@cholcombe973 I've rebased the PR to use `mut self` everywhere, and no references. If ok just go ahead and merge it, otherwise let me know.

Reply:
Yeah that's fine.  I didn't realize you were using a builder pattern with that.

Reply:
I actually forgot to push the updated PR before replying here, sorry :disappointed:.
The fixup described above is in https://github.com/Quyzi/gpt/pull/16 now.